client/login/qr-code-login/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { Card, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import QRCode from 'qrcode.react';
+import qrCenter from 'calypso/assets/images/qr-login/wp.png';
+import GlobalNotices from 'calypso/components/global-notices';
+import Main from 'calypso/components/main';
+import { login } from 'calypso/lib/paths';
+
+import './style.scss';
+
+function QrCodeLogin( { translate, locale } ) {
+	const loginParameters = {
+		locale: locale,
+	};
+	return (
+		<Main className={ classNames( 'qr-code-login' ) }>
+			<GlobalNotices id="notices" />
+			<div className="qr-code-login__form">
+				<h1 className="qr-code-login__heading">
+					{ translate( 'Use QR code to login via the Mobile App' ) }
+				</h1>
+
+				<Card className="qr-code-login__card">
+					<QRCode
+						value="https://apps.wordpress.com/get/?campaign=login-qr-code#token=&data="
```

Comment:
Seems like we have a query string, followed by a hash and something else that resembles a query string. Is this expected?

Reply:
Yes, this part is not decided just yet. 
We need a way to pass the encoded data as well as the token to the mobile app. I removed the weirdness for now.